pkg/kubectl/cmd/util/helpers.go

```diff
@@ -406,3 +406,25 @@ func Merge(dst runtime.Object, fragment, kind string) (runtime.Object, error) {
 	}
 	return out, nil
 }
+
+// DumpReaderToFile writes all data from the given io.Reader to the specified file
+// (usually for temporary use).
+func DumpReaderToFile(reader io.Reader, filename string) error {
+	f, err := os.OpenFile(filename, os.O_WRONLY|os.O_CREATE|os.O_TRUNC, 0600)
+	if err != nil {
+		return err
+	}
+	data, err := ioutil.ReadAll(reader)
```

Comment:
I have no idea if in go we worry about possible buffer overflows. If you know that we don't it's OK, otherwise please batch reads.